src/i_winmusic.c

```diff
@@ -43,6 +50,7 @@ enum
 };
 
 char *winmm_midi_device = NULL;
+int winmm_complevel = COMP_STANDARD;
```

Comment:
Should this default to `COMP_VANILLA`? This is Chocolate Doom after all.

Reply:
Fixed by https://github.com/chocolate-doom/chocolate-doom/pull/1598/commits/48868398eb0bd02cc16ba2d28828ffafbc251876